src/ontology/mondo-ingest.Makefile

```diff
@@ -454,7 +455,7 @@ extract-unmapped-matches: lexmatch/README.md
 ###################################
 #### Lexmatch-combine #######
 ###################################
-lexmatch/all_exact.robot.tsv: $(SCRIPTSDIR)/lexmatch-sssom-compare.py
+lexmatch/all_exact.robot.tsv: $(SCRIPTSDIR)/lexmatch-sssom-compare.py lexmatch/README.md
```

Comment:
## Add missing prereq
Unrelated to the bug fix at hand, but related to the lexmatch pipeline.